backend/src/routes/users.js

```diff
@@ -58,14 +58,14 @@ const deleteUser = async (req, res) => {
 
 const createUser = async (req, res) => {
     try {
-        const user = req.body;
+        let user = req.body;
```

Comment:
I don't think it is wise to update this variable. Instead create a new variable like `const createdUser = await db.users.createUser(user)`.